### PR TITLE
Fix Typo in FAQ Doc

### PR DESCRIPTION
**Pull request by andysnell**
This PR fixes a minor typo in the "Redirected Requests" header of the FAQ document.

### CODE REVIEW

File: docs/faq.rst [hunk 1]
```diff
@@ -130,7 +130,7 @@ setting the ``expect`` request option to ``false``:
     // Disable the expect header on all client requests
     $client = new GuzzleHttp\Client(['expect' => false]);
 
-How can I track a redirected requests?
+How can I track redirected requests?
```
Comment (sagikazarmark) [inline docs/faq.rst]: Can you please align the heading line (= signs) as well?

**Comment by sagikazarmark** [comment]
Thanks a lot! I had on minor comment.

**Comment by andysnell** [comment]
I've aligned the "=" as requested.

**Comment by sagikazarmark** [comment]
Awesome, thanks a lot!

**Pull request merged**